e2e/cypress/integration/search_autocomplete/scroll_spec.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+describe('Autocomplete in the search box - scrolling', () => {
+    const usersCount = 15;
+    const timestamp = Date.now();
+
+    before(() => {
+        // # Create new team for tests
+        cy.apiCreateTeam(`search-${timestamp}`, `search-${timestamp}`).then(({team}) => {
+            // # Create pool of users for tests
+            for (let i = 0; i < usersCount; i++) {
+                cy.apiCreateUser().then(({user}) => {
+                    cy.apiAddUserToTeam(team.id, user.id);
+                });
+            }
+            cy.visit(`/${team.name}/channels/town-square`);
```

Comment:
👍 It works for me.
My last fix also works on my machine.
I'm not sure if problem is related to not fully rendered search bar.
My first try to fix it was `cy.get('#searchBox').should('be.visible').type('from:');` (I assume `should('be.visible')` should guarantee that search bar is rendered) - it didn't fix the problem.

I found two other tests which add timeouts (alternative solution?):
https://github.com/mattermost/mattermost-webapp/blob/f6bf926677f8491d542b30b060c509562f767c11/e2e/cypress/integration/enterprise/guest_accounts/guest_add_spec.js#L81

https://github.com/mattermost/mattermost-webapp/blob/f6bf926677f8491d542b30b060c509562f767c11/e2e/cypress/integration/enterprise/integrations/incoming_webhook_spec.js#L64

Reply:
Sometimes there are re-render happening multiple times in the background. So the first one got by Cypress right after visiting the page might no longer available on succeeding action (like on click).
With 2 given examples, the timeouts there seemed to be not necessary. For me, it worked because of the intermediate function/command between visit and searchbox which were `cy.sendDirectMessageToUser` and `cy.postIncomingWebhook` that gave enough time to fully render the page before acting into the searchbox.